### ISSUE TITLE
Bot created in Composer doesn't show 'Open In Composer' in Azure

### ISSUE THREAD

**Issue by xieofxie** [microsoft/BotFramework-Composer#8485]
## Describe the bug

![image](https://user-images.githubusercontent.com/2876650/127284082-88d96cb5-4bf4-4536-b9c0-6d7787a44d59.png)

should be:

![image](https://user-images.githubusercontent.com/2876650/127284064-e842f370-fff0-4b6e-bef2-9f71787edabe.png)

## Browser

- [ ] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [ ] Windows
- [ ] Ubuntu

**Comment by srinaath** [comment]
@luhan2017 Could you have a look at this ticket?

**Comment by hailiu2586** [comment]
@srinaath, depends on which azure cloud is the repro reported. 
Btw, I don't see repro instruction that is actionable in the issue report

**Comment by hailiu2586** [comment]
Btw, if the bot is created from composer originally, then the issue goes to composer team to investigate. Also, it might be worthwhile to check the version of composer used to create the bot

**Comment by srinaath** [comment]
@hailiu2586 Yep. That makes sense. @xieofxie could you specify the Composer version in which the bot was generated and published. In addition, could you specify the Composer version with which you published the bot as well the region used to publish?
![Screen Shot 2021-07-28 at 2 38 48 PM](https://user-images.githubusercontent.com/13004779/127399774-38b281f9-8874-4669-931f-915897d2fa7c.png)

**Comment by luhan2017** [comment]
@hailiu2586 and @srinaath , I have investigated and found that composer provisioned Azure bot don't have this property: " openWithHint: 'bfcomposer://", But in composer code, we did have this code piece, so I doubt this could be an issue of azure sdk? even I updated https://www.npmjs.com/package/@azure/arm-botservice version to the latest 3.0.0, the azurebot is still not provisioned with the correct value. We may need to update the BotProperties in the arm-botservice. 

![image](https://user-images.githubusercontent.com/32191031/127423441-b0c5aa34-a690-45a0-96d7-72cf1434a620.png)

**Comment by hailiu2586** [comment]
@luhan2017  below is the arm payload the azure portal used to create azurebot

```json
        {
            "type": "Microsoft.BotService/botServices",
            "name": "[parameters('botId')]",
            "kind": "azurebot",
            "apiVersion": "2018-07-12",
            "location": "global",
            "sku": {
                "name": "[parameters('sku')]"
            },
            "tags": "[variables('botTags')]",
            "dependsOn": [],
            "properties": {
                "displayName": "[variables('botDisplayName')]",
                "msaAppId": "[parameters('msAppId')]",
                "openWithHint": "bfcomposer://",
                ...
            }
        }
```

Is the composer code using client js sdk or direct arm put to provision azure bot?
